lib/logstasher.rb

```diff
@@ -117,7 +117,7 @@ def custom_fields=(val)
   def log(severity, msg)
     if self.logger && self.logger.send("#{severity}?")
       event = LogStash::Event.new('@source' => self.source, '@fields' => {:message => msg, :level => severity}, '@tags' => ['log'])
-      self.logger.send severity, event.to_json
+      self.logger << event.to_json + "\n"
```

Comment:
I don't understand why this change was made, it completely breaks native json parsing in logstash without using the json_lines codec.